test/e2e/quadlet_test.go

```diff
@@ -456,6 +456,30 @@ var _ = Describe("quadlet system generator", func() {
 			Expect(current[0]).To(ContainSubstring(expected))
 		})
 
+		It("Should scan and return output for files in subdirectories", func() {
+			dirName := "test_subdir"
+
+			fmt.Println("QuadletDir: " + quadletDir)
```

Comment:
```suggestion
			GinkgoWriter.Println("QuadletDir: " + quadletDir)
```
never write directly to stdout in the ginkgo tests, this will make the output unreadable.